t/t2214-job-manager-hold-limited.t

```diff
@@ -34,6 +34,14 @@ test_expect_success HAVE_JQ 'job-manager: job annotations correct (RSSSS)' '
         jmgr_check_no_annotations $(cat job5.id)
 '
 
+test_expect_success 'job-manager: hold job 3, 5 again (issue #4940)' '
+        flux job urgency $(cat job3.id) 16 &&
+        flux job urgency $(cat job3.id) 16 &&
+        flux job urgency $(cat job3.id) 16 &&
+        flux job urgency $(cat job3.id) 16 &&
+        flux job urgency $(cat job5.id) 16
+'
```

Comment:
should be urgency 0/hold instead of 16?

Reply:
yep! :facepalm: nice catch!

Reply:
Thanks. Fixed that blunder and force pushed the result.